x/claims/genesis.go

```diff
@@ -40,7 +40,11 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, data types.GenesisState) {
 		}
 		k.SetClaimsRecord(ctx, addr, cr)
 
-		sumClaims = sumClaims.Add(cr.InitialClaimableAmount)
+
+		sumClaims = sumClaims.Add(k.GetClaimableAmountForAction(ctx, cr, types.ActionVote, data.Params))
+		sumClaims = sumClaims.Add(k.GetClaimableAmountForAction(ctx, cr, types.ActionDelegate, data.Params))
+		sumClaims = sumClaims.Add(k.GetClaimableAmountForAction(ctx, cr, types.ActionEVM, data.Params))
```

Comment:
LGTM. maybe someone can write a test or test manually here before we merge.

Reply:
logic should add the initial claimable per action, will update